### DISCUSSION TITLE
[ClipBoardCopy] - [add ability to add space between text and button]

### DISCUSSION THREAD

**Discussion by adonispuente** [patternfly/patternfly#5572] [Feature requests]
**Is this a new component or an extension of an existing one?**
ClipboardCopy
**Describe the feature**
The ability to allow seperating the button on clipboard from the text so that it can meet new design mocks. As shown here 
![Screen Shot 2023-04-18 at 3 30 41 PM](https://user-images.githubusercontent.com/60629070/232885187-b1b8292d-1a88-4539-9a84-82db01dd2c2f.png)

**Any other information?**

**Comment by mcoker** [comment]
Depending on the design, you could do this now by inserting non-breaking spaces - [here's an example using `&nbsp;`](https://codesandbox.io/s/icy-resonance-fyzrqg?file=/index.tsx). Though if you want the clipboard copy to separate the text and copy button using a variable amount of space from the overall layout, that isn't a great approach.

We could possibly update the inline-block variant to separate the text and copy icon. Something like this to keep the copy icon on the same line as the text - https://codesandbox.io/s/intelligent-cherry-bklx8e?file=/index.tsx. Or this to have the copy button in the top/right - https://codesandbox.io/s/serene-napier-399f6h?file=/index.tsx

cc @mcarrano @andrew-ronaldson  wdyt?

**Comment by mcarrano** [comment]
@adonispuente can you provide a full page mockup to provide more background? I'd like to understand why this layout is necessary.

**Comment by nicolethoen** [comment]
I am going to convert this to a discussion until we determine if this would be a recommended use case and if we'd want to introduce a new ClipboardCopy style variant. Generally, we avoid doing this to keep cross product design/experience consistent.
